noxfile.py

```diff
@@ -343,27 +343,29 @@ def bump(session: nox.Session):
                 f"Either {fragment_file} must already exist, "
                 "or two positional arguments must be provided."
             )
-    install(session, "antsibull-changelog[toml]", "hatch")
+    # Needs newer antsibull-changelog for hatch version auto-detection support
+    install(session, "antsibull-changelog >= 0.24.0", "hatch")
```

Comment:
The hatch version code is only used when a TOML library is present.
```suggestion
    install(session, "antsibull-changelog[toml] >= 0.24.0", "hatch")
```